server/events/runtime/run_step_runner.go

```diff
@@ -37,6 +37,7 @@ func (r *RunStepRunner) Run(ctx models.ProjectCommandContext, command string, pa
 		"PULL_NUM":                   fmt.Sprintf("%d", ctx.Pull.Num),
 		"PULL_AUTHOR":                ctx.Pull.Author,
 		"USER_NAME":                  ctx.User.Username,
+		"PROJECT_NAME":               ctx.ProjectName,
```

Comment:
let's alphabetize these as well since the list is getting long:
```go
	customEnvVars := map[string]string{
		"ATLANTIS_TERRAFORM_VERSION": tfVersion,
		"BASE_BRANCH_NAME":           ctx.Pull.BaseBranch,
		"BASE_REPO_NAME":             ctx.BaseRepo.Name,
		"BASE_REPO_OWNER":            ctx.BaseRepo.Owner,
		"DIR":                        path,
		"HEAD_BRANCH_NAME":           ctx.Pull.HeadBranch,
		"HEAD_REPO_NAME":             ctx.HeadRepo.Name,
		"HEAD_REPO_OWNER":            ctx.HeadRepo.Owner,
		"PLANFILE":                   filepath.Join(path, GetPlanFilename(ctx.Workspace, ctx.ProjectName)),
		"PROJECT_NAME":               ctx.ProjectName,
		"PULL_AUTHOR":                ctx.Pull.Author,
		"PULL_NUM":                   fmt.Sprintf("%d", ctx.Pull.Num),
		"USER_NAME":                  ctx.User.Username,
		"WORKSPACE":                  ctx.Workspace,
	}
```

Reply:
Variables are now sorted alphabetically.